bootcampspringboot/src/main/java/com/max/bootcampspringboot/api/EmployeeController.java

```diff
@@ -0,0 +1,44 @@
+package com.max.bootcampspringboot.api;
+
+import com.max.bootcampspringboot.api.mapper.ApiEmployeeMapper;
+import com.max.bootcampspringboot.api.model.ApiEmployee;
+import com.max.bootcampspringboot.service.EmployeeService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/employees")
+public class EmployeeController {
+    private final EmployeeService employeeService;
+    public EmployeeController(EmployeeService employeeService) {
+        this.employeeService = employeeService;
+    }
+
+    @GetMapping("/{id}")
+    public ApiEmployee getEmployee(@PathVariable int id){
+        return ApiEmployeeMapper.toApiEmployee(employeeService.getEmployee(id));
+    }
+
+    @GetMapping
+    List<ApiEmployee> getAllEmployees(){
```

Comment:
Decorator fehlt

Reply:
meinst du ("")?

Reply:
Decorator in Java oder Typescript sind die access modifier, also private public protected etc.
Ohne sind die bei Java Default und das will man eig nicht außer bei Tests etc.

Reply:
achso das meinst du :D

Reply:
war bei angular und da sind dekorator ja diese @ Dinger